pages/static/javascripts/authform.js

```diff
@@ -59,4 +59,8 @@ $(document).ready(function(){
     })
     return false;
   });
+
+  if(window.location.href.indexOf('#modal-login') != -1) {
+    $(".md-show").css('visibility', 'visible');
```

Comment:
CSS should not be changed _directly_; CSS classes should be toggled instead. This should be refactored in a next PR.